README.md

```diff
@@ -135,7 +135,7 @@ const unbindListener = counter.subscribe(value => {
 ```
 
 `store.subscribe(cb)` in contrast with `store.listen(cb)` also call listeners
-imminently during the subscription.
+immediately during the subscription.
```

Comment:
Maybe you actually meant `imminently` but in all other places it's `immediately`